src/utils/resolveModule.js

```diff
@@ -0,0 +1,16 @@
+import resolve from 'resolve';
```

Comment:
@alexfedoseev Looks like you found a better way to do the location of bootstrap.